packages/build/src/telemetry/request.js

```diff
@@ -4,24 +4,19 @@ const { argv } = require('process')
 
 const got = require('got')
 
-const { version } = require('../../package.json')
+const DEFAULT_ORIGIN = 'https://api.segment.io/v1'
+const WRITE_KEY = 'dWhlM1lYSlpNd1k5Uk9rcjFra2JSOEoybnRjZjl0YTI6'
```

Comment:
Just to confirm: is this key meant to be confidential? (because this repository is public)

Reply:
No, it's an analytics key essentially, used to match data with our source. The only potential for abuse I guess would be for a bad actor to use this to send us fake data, but I believe segment has mechanisms in place to help us avoid spam 🤔 It's essentially no different of when you would serve a JS doc in the frontend with an analytics key.

If we really wanted to prevent this though, we could try to inject it from the `buildbot`, however I'm not sure it would be worth it.

Reply:
Thanks for the explanation.

We have some precedent about this: the Bugsnag key. It had the same situation, including the fact that Bugsnag has mechanisms to prevent against abuse, and that they recommend putting the client key as public in client code.
See https://github.com/netlify/build/issues/803 and https://github.com/netlify/buildbot/issues/634

Do you think this would apply here too? Or that we should do it as a follow-up work?

I would be curious what @erezrokah thinks about this, given his security background.

Reply:
Merging this for now and we can cover any possible changes in follow up work 👍